<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowEditorDibujo</class>
 <widget class="QMainWindow" name="MainWindowEditorDibujo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Editor de dibujo</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/img/paint.png</normaloff>:/images/img/paint.png</iconset>
  </property>
  <widget class="QWidget" name="container">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item alignment="Qt::AlignTop">
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout" columnstretch="1">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item row="11" column="0">
        <widget class="QLabel" name="label_color_fondo">
         <property name="minimumSize">
          <size>
           <width>28</width>
           <height>29</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Color de fondo</string>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="pbutton_seleccionar">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Seleccionar</string>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/images/img/cursor.png</normaloff>:/images/img/cursor.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="8" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="label_color_linea">
         <property name="minimumSize">
          <size>
           <width>28</width>
           <height>29</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Color de línea</string>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QPushButton" name="pbutton_borrar">
         <property name="toolTip">
          <string>Borrar figura seleccionada</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/images/img/delete.png</normaloff>:/images/img/delete.png</iconset>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QPushButton" name="pbutton_rect">
         <property name="toolTip">
          <string>Rectángulo</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/images/img/rectangle.png</normaloff>:/images/img/rectangle.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="10" column="0">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="7" column="0">
        <widget class="QPushButton" name="pbutton_ovalo">
         <property name="toolTip">
          <string>Óvalo</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/images/img/circle.png</normaloff>:/images/img/circle.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="pbutton_linea">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Línea</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/images/img/linea.png</normaloff>:/images/img/linea.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="13" column="0">
        <widget class="QSpinBox" name="spin_ancho_linea">
         <property name="toolTip">
          <string>Ancho de línea</string>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="14" column="0">
        <widget class="QComboBox" name="combo_tipo_linea">
         <property name="toolTip">
          <string>Tipo de línea</string>
         </property>
         <item>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/images/img/linea.png</normaloff>:/images/img/linea.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/images/img/dashed.png</normaloff>:/images/img/dashed.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/images/img/puntos.png</normaloff>:/images/img/puntos.png</iconset>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="canvas_container">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="accion_guardar"/>
    <addaction name="accion_abrir"/>
    <addaction name="separator"/>
    <addaction name="accion_salir"/>
   </widget>
   <addaction name="menuArchivo"/>
  </widget>
  <action name="accion_guardar">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/img/save.png</normaloff>:/images/img/save.png</iconset>
   </property>
   <property name="text">
    <string>Guardar dibujo</string>
   </property>
  </action>
  <action name="accion_abrir">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/img/open.png</normaloff>:/images/img/open.png</iconset>
   </property>
   <property name="text">
    <string>Abrir dibujo</string>
   </property>
  </action>
  <action name="accion_salir">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/img/exit.png</normaloff>:/images/img/exit.png</iconset>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
